lib/compilers/typescript.js

```diff
@@ -1,3 +1,27 @@
+// Copyright (c) 2022, Compiler Explorer Authors
```

Comment:
@ODuzhar the file you contributed 2 months ago was missing a license. Are you ok with this copyright attribution?

Reply:
sure